.packit.yml

```diff
@@ -0,0 +1,34 @@
+upstream_package_name: subscription-manager
+downstream_package_name: subscription-manager
+specfile_path: subscription-manager.spec
+upstream_tag_template: "subscription-manager-{version}-1"
+
+srpm_build_deps:
+  - python3
+  - tito
+
+actions:
+  pre-sync:
+    - tito build --test --tgz
```

Comment:
this hook is only used when pushing a new release downstream, or pulling a new release from downstream -- see also:
https://packit.dev/docs/configuration/actions#syncing-the-release

since this config is used only for building upstream (there is a separate config in fedora already), this hook is effectively not needed here